src/app/@ansyn/ansyn/modules/menu-items/cases/services/cases.service.ts

```diff
@@ -184,6 +186,16 @@ export class CasesService {
 			);
 	}
 
+	generateUUID(): string {
+		return UUID.UUID();
+	}
+
+	createLink(link): Observable<any> {
+		return this.storageService.create(this.linksConfig.schema, link).pipe(
+			map(_ => link)
```

Comment:
basically, if all you need is the link id,
you can take it from the storage response (e.g. res._id) and remove this map pipe
but if you want to keep it like that is ok(cause we do the same thing on case create)